numpy/f2py/src/fortranobject.c

```diff
@@ -264,7 +266,7 @@ fortran_doc(FortranDataDef def)
         size -= n;
 
         if (def.data == NULL) {
-            n = format_def(p, size, def) == -1;
+            n = format_def(p, size, def);// == -1;
```

Comment:
Might as well remove the comparison as comment it out.

Reply:
Sorry, that was supposed to go when I took the PR out of draft. Done!